src/app/space/wizard/services/fabric8-app-generator.client.mock.ts

```diff
@@ -1,4 +1,4 @@
-import { IAppGeneratorResponse } from "app/space-wizard/models/contracts/app-generator-response";
+import { IAppGeneratorResponse } from "..//models/contracts/app-generator-response";
```

Comment:
good point. this import is invalid and not used. let me clean unused import in wizard.